### PR TITLE
HMR Fix

### PR DESCRIPTION
**Pull request by ForsakenHarmony**
This fixes HMR, not just browser reloads on change

### CODE REVIEW

File: src/lib/webpack/webpack-client-config.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ function isDev(config) {
 			disableHostCheck: true,
 			historyApiFallback: true,
 			quiet: true,
-			clientLogLevel: 'info',
+			clientLogLevel: 'none',
```
Comment (reznord) [inline src/lib/webpack/webpack-client-config.js]: we can be leaving this to `info` right?

Comment (reznord) [reply src/lib/webpack/webpack-client-config.js]: nvm, got the answer in slack...lol

**Comment by reznord** [comment]
Thanks a ton!

File: src/lib/webpack/webpack-client-config.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ function clientConfig(env) {
 	};
 
 	if (!isProd) {
-		entry['hmr'] = `webpack-dev-server/client?http://localhost:${process.env.PORT || env.port || 8080}`;
+		entry.bundle = [
```
Comment (lukeed) [inline src/lib/webpack/webpack-client-config.js]: This needed to be a `concat` or `push`. Currently creating a nested array.

Comment (lukeed) [reply src/lib/webpack/webpack-client-config.js]: Just kidding~!

**Pull request merged**